ecosystem/typescript/sdk_v2/src/utils/hexData.ts

```diff
@@ -0,0 +1,67 @@
+import { bytesToHex, hexToBytes } from "@noble/hashes/utils";
+import { Hex } from "../types";
+
+export class HexData {
+  private hexData: Hex;
```

Comment:
If the class has an internal representation of hex data, let's not use Hex, this is what made HexString so confusing. Can we instead represent hex data using Uint8Array instead, similar to how I do it in AccountAddress?